config-connector/solutions/sql/helm/postgres-ha/README.md

```diff
@@ -0,0 +1,93 @@
+# PostgreSQL High Availability
+
+==================================================
+
+## NAME
+
+  postgres-ha
+
+## SYNOPSIS
+
+  Config Connector compatible yaml files to configure a high availability PostgreSQL cluster.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the service account folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/sql/helm/postgres-ha
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
+1. A working Config Connector instance.
+1. Cloud SQL Admin API enabled in the project where Config Connector is
+      installed
+1. Cloud SQL Admin API enabled in the project managed by Config Connector if
+      it is a different project
```

Comment:
What does the "it" mean in "if it is a different project"? Could you be more specific here?